ocs_ci/ocs/ui/page_objects/virtualmachine_ui.py

```
@@ -579,33 +468,366 @@ def click_actions_control_then_start(self):
+        After saving the Add Volume dialog, wait for the 'Clone in progress'
+        badge to disappear from the boot volume row on the Boot source page.
+        The clone can take up to 7 minutes
+
+        Args:
+            timeout (int): Maximum seconds to wait 15 minutes.
+
+        Raises:
+            TimeoutExpiredError: If the badge is still present after *timeout*.
+        """
+        clone_loc = self.vm_loc["boot_volume_clone_in_progress"]
+        logger.info(f"Waiting up to {timeout} s for 'Clone in progress' to finish...")
```

Ruff (0.16.1)

[warning] Logging statement uses f-string

(G004)